webapp/tbapi/tbapi/blueprints/api.py

```diff
@@ -1,20 +1,138 @@
-from flask import Blueprint, render_template, make_response, jsonify, flash, redirect, url_for, request, current_app
+from flask import Flask, Blueprint, render_template, make_response, jsonify, flash, redirect, url_for, request, current_app, abort
 from tbapi.models.mpower import Patient, User
+from flask_swagger import swagger
+from tbapi.models.tbapp import Note
 
 bp = Blueprint('api', __name__)
 
-@bp.route('/patients')
-def patient_index():
-    return jsonify(patients=[i.serialize for i in Patienst.query.all()])
+@bp.route('/notes/<int:id>', methods=['GET']) # note id
+def getNote(id):
+    if id: # note id
```

Comment:
In the future I might invert this condition, into [a guard clause](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html), then the bulk of the view code will be less nested.
eg:
```
if not id:
    abort(400, 'note id required')

return jsonify(notes=[i.serialize for i in Note.get_by_note(id)])
```